semantic-ui-react-redux/src/main/scala/demo/Main.scala

```diff
@@ -0,0 +1,17 @@
+package demo
+
+import japgolly.scalajs.react.vdom.html_<^._
+import org.scalajs.dom
+import typings.redux.mod.{Store, createStore}
+import typings.reduxDevtoolsExtension.developmentOnlyMod.devToolsEnhancer
+import typings.reduxDevtoolsExtension.mod.EnhancerOptions
+
+import scala.scalajs.js
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    val Store: Store[GithubSearch.State with js.Object, GithubSearch.SearchAction] with Any =
+      createStore(GithubSearch.Reducer, devToolsEnhancer(EnhancerOptions().setName("github search store")))
+    <.h1("react redux").renderIntoDOM(dom.document.getElementById("container"))
```

Comment:
I guess you had problems with `connect` (or the `simpleConnect` facade from the old source code)?

I'll help you revive it if needed, but be aware that `simpleConnect` is only simple in the "does little" sense. It doesn't provide any means of connecting more than one reducer, which you absolutely want to do. 

In fact I left it at that because it's a semi-difficult typing problem in Scala. Not saying there aren't nice solutions, but be prepared to really grok how `connect` actually works if you haven't already :)